exampleSite/content/jobs/melee/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+menu:
+  main:
+    name: Melee
+    parent: melee
+role: melee
+---
+Melee do positional and stuff.
```

Comment:
uhhh, I notice there's a `melee/_index.md` and `melees/_index.md` here.

Reply:
oh shoot! fixing it right away

Reply:
prolly was generated when I was testing and I forgot to delete it.

Reply:
Ok I removed `melees` since we are going with `melee`